training/deepspeech_training/evaluate.py

```diff
@@ -73,7 +73,8 @@ def evaluate(test_csvs, create_model):
 
     loss = tfv1.nn.ctc_loss(labels=batch_y,
                             inputs=logits,
-                            sequence_length=batch_x_len)
+                            sequence_length=batch_x_len,
+                            ignore_longer_outputs_than_inputs=True)
```

Comment:
'm very much against that, since requiring this flag is a symptom of a broken dataset. I understand the rationale about simplicity, but each and every cases I worked on where this was "required" because "trust me the data is fine it's just the code that fails" turned out to end as "trained model is not really efficient" which was effectively fixed after fixing the dataset and removing this flag.

Said otherwise, this 99.99% covers real data issues and make it harder to know there's a problem, leading to a bad model.

Reply:
I understand the philosophy! Will revert!

Reply:
OK, I struck this out!